README.md

```diff
@@ -3,18 +3,39 @@
 ## What is this?
 This explainer outlines the benefits of compression dictionaries, details the different use case for them, and then proposes a way to deliver such dictionaries to browsers to enable these use cases.
 
+## Summary
+This proposal adds support for using designated previous responses, as an external dictionary for [Brotli](https://github.com/google/brotli)-compressing HTTP responses. 
+
+HTTP `Content-Encoding` is extended with a new encoding type and support for allowing responses to be used as dictionaries for future requests. *All actual header values and names still TBD*:
+
+* Server responds to a request for a cacheable resource with an `bikeshed-use-as-dictionary: <scope/path>`.
+* The client will store a SHA-256 hash of the uncompressed response and the applicable `scope` for the resource with the cached response to identify it as a dictionary.
+* On future requests, the client will match a request against an available dictionary with the best-matching `scope`. If a dictionary is available for a given request, the client will add `sbr` to the `Accept-Encoding` request header as well as a `sec-bikeshed-available-dictionary: <SHA-256>` header with the hash of the best available dictionary.
+* If the server has a compressed version of the request URL with the matching dictionary, it serves the dictonary-compressed response with `Content-Encoding: sbr` and `Vary: Accept-Encoding,sec-bikeshed-available-dictionary`.
+
+For interop reasons, `sbr` compression is only supported on secure contexts (similar to brotli compression).
+
+There are also some browser-specific protections independent of the transport compression:
+* Dictionary responses can only be used for resources with the same [fetch destination](https://fetch.spec.whatwg.org/#concept-request-destination) as the dictionary.
+* There are [CORS](https://fetch.spec.whatwg.org/#http-cors-protocol) requirements ([detailed below](#security-mitigations)) for both the dictionary and compressed resource.
+* The `scope` of a dictionary can not refer to paths higher in the directory structure (similar to service worker scope requirements).
```

Comment:
Does that mean that shared dictionaries for dynamic resources would have to be delivered from a relatively high path? (or at least higher than any resource they may be used on)

Reply:
Yes, but that shows that the owner of the dictionary is in control of the path they are trying to take over dictionary rights for.

This is used in service workers for the shared hosting or multiple app case where `/site1` and `/site2` may be controlled by different entities and/or teams and it would be bad if `/site2` installed a dictionary for `/`